ospec/ospec.js

```diff
@@ -84,40 +84,56 @@ module.exports = new function init(name) {
 				if (cursor === fns.length) return
 
 				var fn = fns[cursor++]
+				var timeout = 0, delay = 200, s = new Date
+				var isDone = false
+
+				function done() {
```

Comment:
Technically, yes, but it's basic functionality IMHO. Even Tape supports async errors (although indirectly in that case), and [blue-tape handles promises gracefully and accordingly](https://github.com/spion/blue-tape/blob/master/blue-tape.js).